pkg/client/factory.go

```diff
@@ -253,6 +286,81 @@ func (f *OptimizelyFactory) StaticClient() (optlyClient *OptimizelyClient, err e
 	return optlyClient, err
 }
 
+func (f *OptimizelyFactory) initializeOdpManager(appClient *OptimizelyClient) {
+	// Default value for odp identify call
+	identify := true
+	if f.identify != nil {
+		identify = *f.identify
+	}
+	f.identify = &identify
+	appClient.identify = identify
+
+	if f.optimizelySDKSettings == nil {
+		f.optimizelySDKSettings = &OptimizelySdkSettings{
+			SegmentsCacheSize:          pkgOdpUtils.DefaultSegmentsCacheSize,
+			SegmentsCacheTimeoutInSecs: pkgOdpUtils.DefaultSegmentsCacheTimeout,
+		}
```

Comment:
What is this structuring for? I see `optimizelySDKSettings` is used in this function only.